Project.toml

```
@@ -6,6 +6,12 @@ version = "1.3.5"
 [deps]
 Extents = "411431e0-e8b7-467b-b5e0-f676ba4f2910"
 
+[weakdeps]
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+
+[extensions]
+RecipesBaseExt = "RecipesBase"
+
 [compat]
 Extents = "0.1.1"
 julia = "1"
```

We'd have to bump Julia compat or keep publishing GeoInterfaceRecipes to avoid breakage IMO.

```suggestion
julia = "1.9"
```